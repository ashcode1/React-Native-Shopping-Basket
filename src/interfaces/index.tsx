// Product line item
export interface Item {
  id: number;
  name: string;
  img: string;
  colour: string;
  price: number;
  quantity: number;
  addOnPress?: () => void;
  subtractOnPress?: () => void;
  removeAllOnPress?: () => void;
}

// Colour filter picker item
export interface SelectedColor {
  label: string;
  value: string;
}
