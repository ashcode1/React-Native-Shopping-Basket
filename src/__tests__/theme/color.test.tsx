import * as colors from '../../interfaces/index';

describe('Colors file', () => {
  it('Remains Unchanged', () => {
    expect(colors).toMatchSnapshot();
  });
});
