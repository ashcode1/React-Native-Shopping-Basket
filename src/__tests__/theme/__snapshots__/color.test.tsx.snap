// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Colors file Remains Unchanged 1`] = `Object {}`;
